.github/workflows/request-management-api-ci-v2.yaml

```
@@ -0,0 +1,49 @@
+# GitHub Actions workflow for building and tagging the API image on PR
+name: REQUEST MANAGEMENT API - Continuous Integration
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review] # Triggered by opened or changed pull requests.
+    branches: [main]
+    paths:
+      - 'request-management-api/**' # Triggers on changes to files in the express-api/ directory.
+
+jobs:
+# Job for automated unit tests goes HERE
+
+# Build the docker image, tag with pull request
+  COMPRESSION-SERVICE-IMAGE-BUILD:
+  runs-on: ubuntu-latest
+  steps:
+# Checkout repo
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+# Login to BC Gov Openshift Image Repository via docker
+
+    - name: Login to Openshift Docker
+      run: |
+        docker login ${{ secrets.PUBLIC_IMAGE_REPOSITORY }} -u ${{ secrets.OPENSHIFT_SA_NAME }} -p ${{ secrets.OPENSHIFT_SA_TOOLS_TOKEN }}
+
+# Build the Compression Service Image, Tag with Pull Request number
+
+    - name: Build Compression Image
+      run: |
+        cd request-management-api && docker build -t ${{ secrets.PUBLIC_IMAGE_REPOSITORY }}/${{ secrets.OPENSHIFT_TOOLS_NAMESPACE }}/request-management-api:${{github.event.pull_request.number}} -f Dockerfile.local
```

Small suggestion 🙂

GitHub Actions recommends using working-directory instead of cd for steps. It’s a bit clearer, safer, and will fail fast if the directory doesn’t exist, which helps avoid subtle issues.

Ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsworking-directory

---

fixed